Python/specialize.c

```diff
@@ -215,13 +211,63 @@ print_gc_stats(FILE *out, GCStats *stats)
     }
 }
 
+static void
+print_histogram(FILE *out, const char *name, uint64_t hist[_Py_UOP_HIST_SIZE])
+{
+    for (int i = 0; i < _Py_UOP_HIST_SIZE; i++) {
+        fprintf(out, "%s[%d]: %" PRIu64 "\n", name, (1 << (i + 2)), hist[i]);
```

Comment:
Why `i + 2`? Is it to correct for the `>> 2` when recording the stats?

Oh, I think I see. We're starting with the "third" bucket, since tiny traces are basically nonexistent. Maybe add another constant next to `_Py_UOP_HIST_SIZE` (like `#define _Py_UOP_HIST_BIAS 2` or something) so it's clearer why we're doing this and where.

Reply:
Yes, that's right. Adding the `#define` for bias makes sense.